contracts/coordinator/contracts/test/TestMixins.sol

```diff
@@ -0,0 +1,40 @@
+/*
+
+  Copyright 2018 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.5;
+pragma experimental "ABIEncoderV2";
+
+import "@0x/contracts-exchange-libs/contracts/src/LibEIP712.sol";
+import "../src/libs/LibConstants.sol";
+import "../src/MixinSignatureValidator.sol";
+import "../src/MixinCoordinatorApprovalVerifier.sol";
+
+
+// solhint-disable no-empty-blocks
```

Comment:
This file was previously removed as well, I don't believe it is currently used anywhere.